client/dashboard/app/query-client.ts

```diff
@@ -20,7 +20,10 @@ export const queryClient = new QueryClient( {
 	},
 } );
 
-const persister = createSyncStoragePersister( { storage: window.localStorage } );
+const persister = createSyncStoragePersister( {
+	storage: typeof window !== 'undefined' ? window.localStorage : null,
```

Comment:
Hmmm.... should we just add `@jest-environment jsdom` to the test file instead? (not sure)